portal-wire-protocol.md

```diff
@@ -2,8 +2,9 @@
 
 The Portal wire protocol is the default p2p protocol by which Portal nodes communicate.
 
-The different Portal networks **MAY** use this protocol, but they **MUST** remain separated per network.
-This is done at the [Node Discovery Protocol v5](https://github.com/ethereum/devp2p/blob/master/discv5/discv5-wire.md#talkreq-request-0x05) layer, by providing a different protocol byte string, per network, in the `TALKREQ` message.
+The different protocol within the Portal network **MAY** use this protocol, but they **MUST** remain separated per network.
```

Comment:
```suggestion
The different sub-protocols within the Portal network **MAY** use this wire protocol, but they **MUST** remain separated per network.
```
Something like this ^ could help disambiguate the repeated use of the word protocol with different meanings.